src/docs/statistics.dox

```diff
@@ -74,10 +74,10 @@ when the cursor was opened), as well as statistics logging, when it is
 configured.
 
 The following example configures WiredTiger to maintain only relatively
-inexpensive statistics, and to clears statistics after they are gathered
-or logged:
+inexpensive statistics, and to clear statistics after they are gathered or
+logged:
 
-@snippet ex_all.c Statistics clear configuration
+@snippet ex_all.c Statistics cursor session
```

Comment:
snippet should be "clear configuration" not "cursor session" ?. is my understanding wrong.

Reply:
That's right. Thanks.